scripts/installDepsScripts/installDotnetSDK8.sh

```diff
@@ -50,9 +50,10 @@ if [ "$source" != "https://packages.microsoft.com/$repo/$repo_version/prod" ]; t
     rm packages-microsoft-prod.deb
 
     # Remove the dotnet-sdk installation that doesn't come from the Microsoft source
+    set +e
```

Comment:
Does it makes sense to add also this fix on `installDotnetSDK7` script, which is used by Avalonia templates?

Reply:
yes make sense, added here 3313e68